build.sbt

```diff
@@ -247,6 +242,29 @@ lazy val pubsub = project
     addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1")
   )
   .enablePlugins(BuildInfoPlugin)
+
+lazy val pubsub = project
+  .in(file("modules/pubsub"))
+  .dependsOn(commonFs2)
+  .settings(BuildSettings.basicSettings)
+  .settings(BuildSettings.formatting)
+  .settings(BuildSettings.scoverageSettings)
+  .settings(BuildSettings.sbtAssemblySettings)
+  .settings(
+    name := "snowplow-enrich-pubsub",
+    description := "High-performance streaming Snowplow Enrich job for PubSub built on top of functional streams",
+    buildInfoKeys := Seq[BuildInfoKey](organization, name, version, description),
```

Comment:
Nitpick: I think settings coming from separate plugins (`sbt-buildinfo` in this case should be in a separate list, like `sbtAssemblySettings` or `scoverageSettings`). Not a hard requirement though.